web/src/app/workplace/taskdetails/general/general-fields.component.html

```diff
@@ -0,0 +1,113 @@
+<ng-container *ngIf="task">
+  <div class="col-md-12">
+    <div class="form-group">
+      <label for="task-description" class="control-label">Description</label>
+      <textarea class="form-control" id="task-description" placeholder="Description"
+                [(ngModel)]="task.description"
+                name="task.description"></textarea>
+    </div>
+  </div>
+  <div class="col-md-6">
+    <div class="form-group">
+      <label for="task-owner" class="control-label">Owner</label>
+      <input type="text" class="form-control" id="task-owner" placeholder="Owner" [(ngModel)]="task.owner"
+             name="task.owner">
+    </div>
+    <div class="form-group">
+      <label for="task-priority" disabled class="control-label">Priority</label>
+      <input type="text" class="form-control" id="task-priority" placeholder="no priotity set"
+             [(ngModel)]="task.priority"
+             name="task.priority">
+    </div>
+    <div class="form-group">
+      <label for="task-state" class="control-label">State</label>
+      <input type="text" disabled class="form-control" id="task-state" placeholder="Task has no State"
+             [(ngModel)]="task.state"
+             name="task.state">
+    </div>
+    <div class="form-group">
```

Comment:
swap with state this way we will have all readonly fields ordered.

Reply:
done